src/components/Flipper/FlipperFilter/FlipperFilter.tsx

```diff
@@ -59,52 +63,62 @@ function FlipperFilter(props: Props) {
+    }
+
+    const countdownRenderer = ({ minutes, seconds }) => (
+        <span>
+            {zeroPad(minutes)}:{zeroPad(seconds)}
+        </span>
+    );
+
+    const nonPremiumTooltip = <span>This is a premium feature.<br />(to use for free wait  <Countdown onComplete={() => onFreePremium()} renderer={countdownRenderer} date={FREE_PREMIUM_FILTER_TIME} />)</span>;
+
     const binFilter = <Form.Group>
         <Form.Label htmlFor="onlyBinCheckbox" className="flipper-filter-formfield-label only-bin-label">Only BIN-Auctions</Form.Label>
-        <Form.Check id="onlyBinCheckbox" onChange={onOnlyBinChange} className="flipper-filter-formfield" type="checkbox" disabled={!props.isPremium} />
+        <Form.Check id="onlyBinCheckbox" onChange={onOnlyBinChange} className="flipper-filter-formfield" type="checkbox" disabled={!props.isPremium&& !freePremiumFilters} />
 
     </Form.Group>;
     const soldFilter = <Form.Group>
         <Form.Label htmlFor="onlyUnsoldCheckbox" className="flipper-filter-formfield-label only-bin-label">Hide SOLD Auctions</Form.Label>
-        <Form.Check id="onlyUnsoldCheckbox" onChange={onOnlyUnsoldChange} defaultChecked={props.isPremium} className="flipper-filter-formfield" type="checkbox" disabled={!props.isPremium} />
+        <Form.Check id="onlyUnsoldCheckbox" onChange={onOnlyUnsoldChange} defaultChecked={props.isPremium} className="flipper-filter-formfield" type="checkbox" disabled={!props.isPremium && !freePremiumFilters} />
     </Form.Group>;
 
     const numberFilters = <div>
         <Form.Group style={{ width: "45%", display: "inline-block" }}>
             <Form.Label className="flipper-filter-formfield-label">Min Profit:</Form.Label>
-            <Form.Control onChange={onMinProfitChange} className="flipper-filter-formfield" type="number" step={5000} disabled={!props.isLoggedIn} />
+            <Form.Control onChange={onMinProfitChange} className="flipper-filter-formfield" type="number" step={5000} disabled={!props.isPremium && !freePremiumFilters} />
         </Form.Group>
         <Form.Group style={{ width: "45%", display: "inline-block", marginLeft: "5%" }}>
             <Form.Label className="flipper-filter-formfield-label">Max Cost:</Form.Label>
-            <Form.Control onChange={onMaxCostChange} className="flipper-filter-formfield" type="number" step={20000} disabled={!props.isLoggedIn} />
+            <Form.Control onChange={onMaxCostChange} className="flipper-filter-formfield" type="number" step={20000} disabled={!props.isPremium && !freePremiumFilters} />
         </Form.Group>
     </div>;
 
-
-    function updateOnlyUnsold(isActive: boolean) {
-        setOnlyUnsold(isActive);
-        let filter = getCurrentFilter();
-        filter.onlyUnsold = isActive;
-        props.onChange(filter);
-    }
-
     return (
         <div>
             <Form style={{ marginBottom: "5px" }} >
-                {!props.isLoggedIn ?
+                {!props.isPremium && !freePremiumFilters ?
```

Comment:
This should still require a login and/or display that login is an alternative to unlock it